Gems/Atom/RPI/Code/Source/RPI.Reflect/Material/MaterialAssetCreator.cpp

```diff
@@ -11,6 +11,14 @@
 #include <Atom/RPI.Reflect/Image/AttachmentImageAsset.h>
 #include <AzCore/std/sort.h>
 #include <AzCore/std/bind/bind.h>
+#include <AzCore/Console/Console.h>
+
+AZ_CVAR(bool,
+    r_enableMaterialPropertyNames,
+    true,
+    nullptr,
+    AZ::ConsoleFunctorFlags::Null,
+    "Enables the use material property names in material assets, breaking its AP job dependency on material type assets.");
```

Comment:
Ah, another reason we shouldn't use a cvar like this: this will affect all materials in the project, which  is too wide a brush. We can have the MaterialAssetBuilderComponent include the names for FBX, and the MaterialBuilder omit the names for all other cases. In the future, we could even change the setting on a per-material basis.

Reply:
Instead, I suggest passing the setting into MaterialAssetCreator::Begin

Reply:
Oh good point on being able to change the setting on a per-material basis!